domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/util/PublishFieldsValidator.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.ccd.definition.store.domain.validation.util;
+
+
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.eventcasefield.EventCaseFieldEntityValidationContext;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.EventCaseFieldEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.EventComplexTypeEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.EventEntity;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public interface PublishFieldsValidator {
+
+    String PUBLISH_AS_COLUMN = "PublishAs";
+    String PUBLISH_COLUMN = "Publish";
```

Comment:
you should use the values in the `ColumnName` enum

Reply:
domain gralde module has PublishFieldsValidator class and  excel-importer ColumnName. There is not relation between domain and excel-importer .   For that reason I cannot use ColumnName.